### PR TITLE
small tfix in cmd factory comment

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

**What this PR does / why we need it**:
tfix in cmd factory comment

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/util/factory.go [hunk 1]
```diff
@@ -240,7 +240,7 @@ type ObjectMappingFactory interface {
 	OpenAPISchema() (openapi.Resources, error)
 }
 
-// BuilderFactory holds the second level of factory methods.  These functions depend upon ObjectMappingFactory and ClientAccessFactory methods.
+// BuilderFactory holds the third level of factory methods. These functions depend upon ObjectMappingFactory and ClientAccessFactory methods.
```
Comment (xiangpengzhao) [inline pkg/kubectl/cmd/util/factory.go]: Why is `third` here? I must be missing something :)

Comment (Mashimiao) [reply pkg/kubectl/cmd/util/factory.go]: ClientAccessFactory is the first level.
ObjectMappingFactory depends on ClientAccessFactory and it's the second level.
BuilderFactory depends on ObjectMappingFactory, so I think it should be the third level.

Comment (shiywang) [reply pkg/kubectl/cmd/util/factory.go]: yes, rings are `ClientAccessFactory -> ObjectMappingFactory -> BuilderFactory`, thanks for the comment fix

**Comment by shiywang** [comment]
/lgtm
ping @adohe for approval

**Comment by Mashimiao** [comment]
ping @adohe

**Comment by Mashimiao** [comment]
ping @adohe

**Comment by Mashimiao** [comment]
ping @adohe

**Pull request merged**